x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -32,6 +31,7 @@ import {
   MAX_BULK_GET_CASES,
   MAX_CASES_PER_PAGE,
 } from '../../constants';
+import { CaseConnectorRt } from '../../types/domain/connector/v1';
```

Comment:
Are we using `v1` here because this file will be the future `types/domain/cases/v1` and their types are tied together? Did I get that right?

Reply:
Yes, this is the reason. It will change very soon when the case will be versioned.